include/dba_pdo.php

```diff
@@ -147,15 +146,13 @@ public function q($sql, $onlyquery = false) {
 		$a->save_timestamp($stamp1, "database");
 
 		/// @TODO really check $a->config for 'system'? it is very generic and should be there
```

Comment:
Yes, it has to be checked that way because the regular method would contain a database access - so this would end in an endless loop. Means: You can remove that comment.